run.py

```diff
@@ -1,11 +1,13 @@
+from flasgger import Swagger
+
 from app import create_app
 
 app = create_app('config.DevelopmentConfig')
-
+swagger = Swagger(app)
 
 @app.route('/')
 def index():
-    return "<h1>Hello, world</>"
+    return '<h1>Hello, world</>'
 
 
 if __name__ == '__main__':
```

Comment:
There is no need for this file if you have another script manager i.e.`manage.py`.
Let all your scripts be handled by your script manager.